src/LoRa.h

```diff
@@ -57,9 +57,11 @@ class LoRaClass : public Stream {
 
 #ifndef ARDUINO_SAMD_MKRWAN1300
   void onReceive(void(*callback)(int));
+  void onCadDone(void(*callback)(boolean));
```

Comment:
Alternative name suggestions: `onCollisionActivityDetectComplete`?

Reply:
I believe CAD in Semtech LoRa is Channel Activity Detection.
There is someone sending the data. and it may be not a collision, right?

In page 44, it says "Once the calculation is finished the modem generates the CadDone interrupt. If the correlation was successful, CadDetected is generated simultaneously."

So the name should be onCadDone or onCadDetected.
My 2 cents.

Reply:
But those interrupts are generated on different pins, if you check Table 18, CadDone is raised on DIO0 & DIO3 and CadDetected is raised on DIO1. They also have different functions. CAD is just the process to see if there is any activity on the channel, CadDone raised when it's done. CadDetect is only fired if activity is detected.

Reply:
Then onCadDone is fine.
As channel activity doesnt imply channel collission.
When there is channel avtivity and we are sending other packets, then there is collission.

Reply:
Prefer onCadDone since it's quite simpler  than on24Done. :-)